--- origsrc/gxine-0.5.907/src/console_output.h	2010-04-11 09:46:54.000000000 -0500
+++ src/gxine-0.5.907/src/console_output.h	2013-01-17 00:34:16.024398000 -0600
@@ -21,7 +21,7 @@
 #ifndef GXINE_CONSOLE_OUTPUT_H
 #define GXINE_CONSOLE_OUTPUT_H
 
-#include <glib/gmessages.h>
+#include <glib.h>
 
 void console_output_init (void);
 
--- origsrc/gxine-0.5.907/src/engine.c	2011-10-11 17:47:52.000000000 -0500
+++ src/gxine-0.5.907/src/engine.c	2013-01-17 00:34:26.915280900 -0600
@@ -30,7 +30,7 @@
 #include <errno.h>
 #include <unistd.h>
 
-#include <glib/gthread.h>
+#include <glib.h>
 
 #include "engine.h"
 #include "ui.h"
--- origsrc/gxine-0.5.907/src/main.c	2010-12-05 05:38:44.000000000 -0600
+++ src/gxine-0.5.907/src/main.c	2013-01-17 00:34:41.113083800 -0600
@@ -46,7 +46,6 @@
 #include <gdk/gdk.h>
 #include <gdk/gdkkeysyms.h>
 #include <glib.h>
-#include <glib/gmessages.h>
 
 #include "desktop_integration.h"
 #include "engine.h"
