--- origsrc/gxine-0.5.903/configure.ac	2008-04-25 16:36:08.000000000 -0500
+++ src/gxine-0.5.903/configure.ac	2008-06-12 23:06:11.156250000 -0500
@@ -324,8 +324,8 @@
 AC_SUBST(CPPFLAGS)
 AC_SUBST(LDFLAGS)
 
-HAVE_MPEG="`test -f "$xine_plugin_dir/xineplug_dmx_mpeg_elem.$acl_cv_shlibext" && echo yes || echo no`"
-HAVE_IMAGE="`test -f "$xine_plugin_dir/xineplug_dmx_image.$acl_cv_shlibext" && echo yes || echo no`"
+HAVE_MPEG="`test -f "$xine_plugin_dir/xineplug_dmx_mpeg_elem$shrext_cmds" && echo yes || echo no`"
+HAVE_IMAGE="`test -f "$xine_plugin_dir/xineplug_dmx_image$shrext_cmds" && echo yes || echo no`"
 
 AC_MSG_CHECKING([which logo format to use])
 echo "$as_me:$LINENO: xine plugin directory: $xine_plugin_dir" >&5
--- origsrc/gxine-0.5.903/src/desktop_integration.c	2008-05-23 13:42:20.000000000 -0500
+++ src/gxine-0.5.903/src/desktop_integration.c	2008-06-12 22:52:21.890625000 -0500
@@ -37,12 +37,18 @@
 #include <sys/stat.h>
 #include <unistd.h>
 #include <errno.h>
+#include <gmodule.h>
 
 #include "desktop_integration.h"
 #include "utils.h"
 
+#ifdef __CYGWIN__
+#define LIBGNOMEVFS "cyggnomevfs-2-0.dll"
+#define LIBGNOMEUI  "cyggnomeui-2-0.dll"
+#else
 #define LIBGNOMEVFS "libgnomevfs-2.so.0"
 #define LIBGNOMEUI  "libgnomeui-2.so.0"
+#endif
 
 typedef enum { GNOME_VFS_OK } GnomeVFSResult; /* incomplete */
 
@@ -494,7 +500,7 @@
 
   section = "Mozilla";
 
-  sname = g_build_filename (plugindir, "gxineplugin.so", NULL);
+  sname = g_build_filename (plugindir, "gxineplugin." G_MODULE_SUFFIX, NULL);
   if (stat (sname, &st))
     return;
 
@@ -511,7 +517,7 @@
     }
     g_free (fname);
 
-    fname = g_build_filename (g_get_home_dir(), dir[i], "plugins/gxineplugin.so", NULL);
+    fname = g_build_filename (g_get_home_dir(), dir[i], "plugins/gxineplugin." G_MODULE_SUFFIX, NULL);
     buf[0] = 0;
 
     /* If the destination object exists, check if it's a symlink and if it's
--- origsrc/gxine-0.5.903/src/wizards.c	2008-06-11 19:35:50.000000000 -0500
+++ src/gxine-0.5.903/src/wizards.c	2008-06-12 22:52:21.921875000 -0500
@@ -32,6 +32,7 @@
 #include <gtk/gtk.h>
 #include <gdk/gdk.h>
 #include <glib.h>
+#include <gmodule.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
@@ -227,7 +228,7 @@
 
   cb.mailcap = new_toggle (vb, "~_/.mailcap", TRUE);
 
-  sname = g_build_filename (plugindir, "gxineplugin.so", NULL);
+  sname = g_build_filename (plugindir, "gxineplugin." G_MODULE_SUFFIX, NULL);
   cb.moz = new_toggle (vb, _("_Mozilla & Mozilla Firefox (plugin)"), !stat (sname, &st));
   g_free (sname);
 
